test/index.js

```diff
@@ -1,6 +1,6 @@
-const assert = require( 'assert' );
-const sinon = require( 'sinon' );
-const memoize = require( '../' );
+import assert from 'assert'
+import sinon from 'sinon'
+import memoize from '../index.js'
```

Comment:
I suspect ESLint may not be testing this file, since I'd expect semicolons.

```suggestion
import assert from 'assert';
import sinon from 'sinon';
import memoize from '../index.js';
```

Reply:
Is the rule of semicolons something that's typically enforced by `prettier` in your ESLint config. I can add `prettier` in, but right now I don't see ESLint errors about the semis/